build/common/installer/scripts/tomlparser.rb

```diff
@@ -25,6 +25,9 @@
 @adxDatabaseName = "containerinsights" # default for all configurations
 @logEnableMultiline = "false"
 @stacktraceLanguages = "go,java,python,dotnet"
+@logEnableKubernetesMetadata = false
+@logKubernetesMetadataiIncludeFields = "podLabels,podAnnotations,podUid,image"
```

Comment:
typo. There extra **i**. should be "logKubernetesMetadataIncludeFields"

Reply:
changed